packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -373,8 +375,8 @@ function TokensPanel({
                 setErrorMessage('')
                 setNewToken(e.target.value)
               }}
-              placeholder="Search by token name, symbol, L1 or L2 address"
-              className="h-full w-full p-2 text-sm font-light text-dark placeholder:text-gray-dark"
+              placeholder={`Search by token name, symbol, ${parentLayer} or ${layer} address`}
+              className="h-full w-full p-2 text-sm font-light text-dark placeholder:text-xs placeholder:text-gray-dark"
```

Comment:
Making placeholder `text-xs` because if `Orbit` word is present the sentence cuts off